lib/codit/population/population.py

```diff
@@ -73,7 +73,7 @@ def realized_r0(self, max_chain_len=4):
         :return: We look at early infectees only.
         """
         n_victims = [len(person.victims) for person in self.people if
-                     person.infector is not None and
+                     len(person.infectors) and
```

Comment:
If infectors is python container, 
```if person.infectors``` rather than ```if len(person.infectors)``` if more pythonic.

because all python containers evaluate to False when they're empty.